zos_concepts/data_transfer/archive_copy_unarchive_restore/archive-fetch.yml

```diff
@@ -0,0 +1,99 @@
+###############################################################################
+# © Copyright IBM Corporation 2023
+###############################################################################
+
+###############################################################################
+# This playbook demonstrates how to use zos_archive to archive a set of data sets  in z/OS
+# and zos_fetch to get the archive to the controller using Red Hat Ansible Certified Content for IBM Z.
+#
+# Usage:
+#  ansible-playbook -i <inventory> <playbook>
+#
+# Example:
+#  ansible-playbook -i inventories archive-fetch.yml
+#
+#
+# Requirements:
+#   IBM z/OS core collection 1.7.0 or later.
+#   Ansible Community General Collection
+#
+#   The ansible.cfg file for this playbook contains the setting stdout_callback = unixy
+#   which changes the format of the playbook output. It has been set to better demonstrate
+#   the effects of whitespace (as compared to the default output formatting).
+#   The unixy callback is part of the community.general package (which is maintained
+#   by the Ansible community).#
+#
+###############################################################################
+
+- hosts: source
+  collections :
+    - "ibm.ibm_zos_core"
+  gather_facts: false
+  environment: '{{ environment_vars }}'
+
+  tasks:
+    - name: Raise error is data_sets_to_archive is not set
```

Comment:
Minor grammar: should be "Raise error if data...."

Reply:
done